scipy/special/tests/test_data.py

```diff
@@ -430,6 +430,8 @@ def test_boost():
         data(spherical_jn_, 'sph_bessel_data_ipp-sph_bessel_data', (0,1), 2, rtol=1e-13),
         data(spherical_yn_, 'sph_neumann_data_ipp-sph_neumann_data', (0,1), 2, rtol=8e-15),
 
+        data(owens_t, 'owenst_data_ipp-owenst', (0, 1), 2, rtol=5e-1),
```

Comment:
I found difference between between real result and result  of `owens_t` function. 
Part of results:
Sample number |  Algorithm | Difference
10 T4 -0.00645986404905
11 T4 -0.00674801958045
12 T4 -0.00772143934778
13 T4 -0.00782197442148
14 T4 -0.00838921922808
15 T4 -0.0084809175162
16 T4 -0.00851552649868
17 T4 -0.00852995038951
18 T4 -0.00881967499371
19 T4 -0.00886193796425
20 T4 1.08420217249e-19
21 T4 2.16840434497e-19
22 T4 -2.16840434497e-19
23 T4 2.16840434497e-19
24 T4 2.16840434497e-19
25 T4 2.16840434497e-19

One algorithm gives difference between real results and function's result from ~0.1e-2 to ~1e-20.

Possible solution: implement all additional algorithm from http://www.boost.org/doc/libs/1_52_0/boost/math/special_functions/owens_t.hpp